peripherals/can/can_c1.yaml

```diff
@@ -0,0 +1,249 @@
+# Copyright 2021 The gd32-rs authors.
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
```

Comment:
This file looks very similar to `can_common.yaml`. Rather than duplicating it all, please include `can_common.yaml` here and then keep just the different parts in this file. If there are parts of `can_common.yaml` which don't apply then you can split them out to a new file which is included for the other models.